src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Queue/NewAzureServiceBusQueue.cs

```diff
@@ -23,129 +23,99 @@ namespace Microsoft.Azure.Commands.ServiceBus.Commands.Queue
         [ValidateNotNullOrEmpty]
         public int? MaxDeliveryCount { get; set; }
 
-        [Parameter(Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "MaxSizeInMegabytes - the maximum size of the queue in megabytes, which is the size of memory allocated for the queue.")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "MaxSizeInMegabytes - the maximum size of the queue in megabytes, which is the size of memory allocated for the queue.")]
         [ValidateNotNullOrEmpty]
         public long? MaxSizeInMegabytes { get; set; }
 
-        [Parameter(Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "MessageCount - the number of messages in the queue.")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "MessageCount - the number of messages in the queue")]
         [ValidateNotNullOrEmpty]
         public long? MessageCount { get; set; }
 
-        [Parameter(Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "RequiresDuplicateDetection - a value that indicates whether the queue supports the concept of session")]
-        [ValidateSet("TRUE", "FALSE",
-            IgnoreCase = true)]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "RequiresDuplicateDetection - a value that indicates whether the queue supports the concept of session")]
+        [ValidateSet("TRUE", "FALSE", IgnoreCase = true)]
         [ValidateNotNullOrEmpty]
         public bool? RequiresDuplicateDetection { get; set; }
 
-        [Parameter(Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "RequiresSession - the value indicating if this queue requires duplicate detection.")]
-        [ValidateSet("TRUE", "FALSE",
-            IgnoreCase = true)]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "RequiresSession - the value indicating if this queue requires duplicate detection")]
+        [ValidateSet("TRUE", "FALSE", IgnoreCase = true)]
         [ValidateNotNullOrEmpty]
         public bool? RequiresSession { get; set; }
 
-        [Parameter(Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "SizeInBytes - the size of the queue in bytes.")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "SizeInBytes - the size of the queue in bytes")]
         [ValidateNotNullOrEmpty]
         public long? SizeInBytes { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Queue/Topic name to forward the messages")]
+        [ValidateNotNullOrEmpty]
+        
```

Comment:
remove space. Also, are you setting these values anywhere?

Reply:
resolved